### ISSUE TITLE
Pure CSS imports

### ISSUE THREAD

**Issue by timarney** [cds-snc/node-starter-app#34]
The imports for purecss are failing
```
/* Base styling from pure.css: https://purecss.io/ */
@import '~purecss/build/base.css';
@import '~purecss/build/buttons-core.css';
@import '~purecss/build/tables.css';
```

<img width="742" alt="Screen Shot 2019-09-17 at 8 17 37 AM" src="https://user-images.githubusercontent.com/62242/65040979-b2e5d400-d923-11e9-9e70-6ee20ebf168e.png">

Pulled in via

Package.json
 "purecss": "^1.0.1",

I'm actually not sure if we still need these styles.

cc: @dsamojlenko @pcraig3

**Comment by dsamojlenko** [comment]
Noticed this yesterday ... guess this came up with moving the sass files out of the public directory ... not sure why - @import should be resolved/rendered by node-sass-middleware, I thought...

Anyway, looking at them, we can probably do without - although I was thinking the normalize.css stylesheet from base might be worth including.

**Comment by dsamojlenko** [comment]
Turns out the `@import` path was wrong (`~` doesn't seem to resolve to `node_modules` in `node-sass-middleware`?)

But, I think we can do without the purecss dependency. I removed it and didn't seem to have any visible effect.

I thought normalize.css might be worth including, since it just evens out browser-rendering differences.

**Comment by dsamojlenko** [comment]
resolved in #35
